### DISCUSSION TITLE
빌드잇 제작월드 이전관련 도움 요청드립니다!

### DISCUSSION THREAD

**Discussion by PXXCASO** [naverz/zepeto-studio-kor#1054] [5. Q&A]
안녕하세요, 폴리북스에서 제페토 월드맵 제작을 의뢰하여, 빌드잇으로 제 계정으로 제작을 해놓았는데, 폴리북스에 월드맵을 이관하려면 어려움이 따른다는 점을 알게 되었습니다. 

제페토 안내 사항에 따르면, 제 개인 계정으로 빌드잇으로 제작한 맵을 유니티로 불러들인 다음, --> 폴리북스 계정으로 제페토 스튜디오를 만들어 그곳에 옮겨보려고 하는데요, 

이것이 프로그램 적으로 가능한 작업인지 궁금합니다. 

안내에 따르면 유니티내에서 zepeto.builit.loader를 설치 후 로드해야하는 것 같은데, 
빌드잇프로그램으로 생성된 파일이 아무것도 없어서 혼선이 있습니다. 

제가만든 빌드잇 맵을 심사승인을 거쳐 공개가 되어야만, 유니티에서 활용할 수 있는 소스가 생기는 것인지요?

 (행사, 홍보용 목적으로 이용시 고객센터에 증빙자료 첨부해야하는 것은 알고있습니다.)  도움 요청드립니다!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 다음과 같은 방법을 안내 드립니다.

1. PXXCASO 님 계정으로 유니티 제페토 계정을 로그인합니다.
2. 빌드잇 월드를 유니티 zepeto.builit.loader 패키지를 통해 로딩합니다.  https://studio.zepeto.me/guides/loading-buildit-maps
- 이 시점에서 빌드잇 오브젝트들이 유니티의 오브젝트로 들어오게 됩니다. 프로젝트 저장을 해 주세요.
3. 유니티에서 계정을 로그아웃하고, 폴리북스 계정으로 로그인합니다. 
4. 제페토 스튜디오에는 폴리북스 계정으로 월드 등록을 해주시고, world id 세팅을 전부 폴리북스 계정에 맞춰서 완료해 주세요. https://studio.zepeto.me/guides/preparing-for-world-creating
5. 이후 원하시는 월드 제작 후 출시 

이 방법으로 잘 되지 않을 경우 export package로 오브젝트를 옮기는 방법도 있습니다.
감사합니다.

**Reply by PXXCASO** [reply]
설명 감사합니다 ^^ 많은도움이 되었습니다.
